### ISSUE TITLE
Bitcode error when archiving

### ISSUE THREAD

**Issue by NickBack** [eppz/Unity.Library.eppz.Cloud#1]
We're using your library for iCloud functionality in our game. When we were Archiving, XCode was giving us the following error:

"bitcode bundle could not be generated because '/Users/KickBack/Desktop/Builds/200/Libraries/Plugins/eppz/eppz! Cloud.a(EPPZ_Cloud_C++.o)' was built without full bitcode. All object files and libraries for bitcode must be generated from Xcode Archive or Install build for architecture armv7"

We tried a variety of things, but the final solution was to go to your cloud project, go to Build Settings, add a User Defined Settings called "BITCODE_GENERATION_MODE" and set it to "bitcode"

Hope that helps someone else as well, and maybe you can update the included .a file in the github project.

**Comment by QuentinArragon** [comment]
Hello NickBack

I'm facing the same "Bitcode Error" problem, and I don't understand your solution. Where exactly you can add a "User Defined Settings" in Build Settings ? Are you talking of an Unity project ?

Many thanks :)

**Comment by NickBack** [comment]
Hey,

No you need to open the source of the plugin in XCode (it's the 3rd link in the repository), then follow my instructions, then you will have generated your own eppz! Cloud.a file and you use that in your Unity project.

Then it should be working fine.

Sorry for not being more detailed, but I did this a while ago and it's from the top of my head,

**Comment by QuentinArragon** [comment]
Thanks for your very fast answer !
But just to be sure (I don't want to bother you too much) : when I add the User Defined Setting, the .a file is automatically updated ? I don't have to go somewhere and press a kind of button "Build .a file" ?

**Comment by NickBack** [comment]
No, you need to build and archive it, so you can get a new .a file.
